src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/ApplicationTest.java

```diff
@@ -53,7 +55,6 @@ void runErrorProcessingFalseTest() {
 
         verify(taskScheduler, times(1)).scheduleWithFixedDelay(mainMessageConsumer,1L);
         verify(taskScheduler, times(1)).scheduleWithFixedDelay(retryMessageConsumer, RETRY_THROTTLE_SECONDS * 1000L);
-
-        //ToDo Ensure error consumer is not called
+        verify(taskScheduler, times(0)).scheduleWithFixedDelay(eq(errorMessageConsumer), anyLong());
```

Comment:
leave this one as is though, as we're checking its not called at all, with any value